ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -213,6 +223,63 @@ final class CommandsValidator(ledgerId: LedgerId) {
     } yield Submitters(actAs, readAs)
   }
 
+  private def validateDisclosedContracts(
+      disclosedContracts: Seq[ProtoDisclosedContract]
+  )(implicit
+      contextualizedErrorLogger: ContextualizedErrorLogger
+  ): Either[StatusRuntimeException, Set[DisclosedContract]] =
+    disclosedContracts.foldLeft[Either[StatusRuntimeException, Set[DisclosedContract]]](
+      Right(Set.empty[DisclosedContract])
+    )((contractz, contract) => {
```

Comment:
Good question 🙂  I just copied the naming convention used in `validateInnerCommands` in the same file. There the strange spelling was used to avoid shadowing the function parameter, but I see this doesn't apply here.